api/internal.yaml

```
@@ -25,12 +25,256 @@ paths:
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopOriginalURL'
+        - name: x-pagopa-lollipop-auth-jwt
+          in: header
+          description: Auth token to retrieve the SPID/CIE User identity from IO
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopJWTAuthorization'
+        - name: x-pagopa-lollipop-assertion-ref
+          in: header
+          description: The id of the assertion to retrieve
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopAssertionRef'
+        - name: x-pagopa-lollipop-assertion-type
+          in: header
+          description: The assertion type (SAML / OIDC)
+          required: true
+          schema:
+            $ref: '#/components/schemas/AssertionType'
+        - name: x-pagopa-lollipop-user-id
+          in: header
+          description: User's fiscal code.
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopUserId'
+        - name: x-pagopa-lollipop-public-key
+          in: header
+          description: User's public key
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopPublicKey'
+        - name: signature-input
+          in: header
+          description: The signature input, needed to verify the `signature` header
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopSignatureInput'
+        - name: signature
+          in: header
+          description: >-
+            The signature of the HTTP request, signed by the client with its
+            private key.
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopSignature'
+      responses:
+        '200':
+          description: Success response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FastLoginResponse'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequest'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Unauthorized'
+        '500':
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalServerError'
 components:
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-Functions-Key
   schemas:
+    InternalServerError:
+      description: InternalServerError
+      allOf:
+        - $ref: '#/components/schemas/ProblemDetail'
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum:
+                - 500
```

should be more compact with this notation if you think that is no more unreadable.
```suggestion
              enum: [500]
```

---

the notation you suggested is already used in the template file. however the codegen generates that field as you see.